app/controllers/moderation/meetings_controller.rb

```diff
@@ -38,7 +40,9 @@ def index
               row.push meeting.title
               row.push meeting.address
               row.push meeting.attendee_count
+              row.push meeting.interventions
               row.push meeting.organization_count
+              row.push meeting.proposals.count
```

Comment:
@josepjaume I added a `counter_cache` but I discovered a nice rails bug: https://github.com/rails/rails/issues/7618 It wasn't solved on Rails 3.1...neither in 4.2... maybe in 5?